consensus/consensus_leader_msg.go

```diff
@@ -42,7 +42,7 @@ func (consensus *Consensus) constructPreparedMessage() ([]byte, *bls.Sign) {
 	}
 
 	consensusMsg := message.GetConsensus()
-	consensus.populateMessageFields(consensusMsg)
+	consensus.populateMessageFields(consensusMsg, consensus.LeaderPubKey)
```

Comment:
Why put as a separate field. Isn't populateMessageFields already have access to consensus.LeaderPubKey?

Reply:
Also used by non-leader during `constructPrepareMessage` which needs non-leader pub key to be populated.